tools/check-swagger

```diff
@@ -1,9 +1,46 @@
 var SwaggerParser = require('swagger-parser');
 
+function checkDuplicateOperationIds(api) {
+    function isDuplicateElement(array, item) {
```

Comment:
Oh, that's a nice simplification! Good catch. I'm going to directly remove the function, since it's a one-liner.

About the snake_case, the only reason for not using it is that today morning I have been working in zulipbot, and my head just messed up repo conventions. Fixed!